### ISSUE TITLE
Migration of files to new folder structure not working

### ISSUE THREAD

**Issue by jmfarina** [danmunn/redmine_dmsf#750]
Hello,
We're using version devel-1.6.0 and it seems that the migration of the files to the new structure isn't working.
I've run `bundle exec rake redmine:plugins:migrate RAILS_ENV=production` and while the DB seems to be expecting the new structure, files remain under the old one (newly uploaded files are stored correctly). I couldn't find a log to check if it reported any problems.

(I commented this in #543 previously, but I didn't realize the issue was closed.)

**Comment by picman** [comment]
Hmm... That's strange. I've recently updated the migration _20170418104901_migrate_documents.rb_ responsible for existing documents migration in order to report any problem directly into the migration output. Anyway, watching the production log during the migration makes sense too.

**Comment by picman** [comment]
You can try this:
1. Back up your database
2. Update the plugin (git pull)
3. Rollback to the last migration before documents migration
```bash
bundle exec rake redmine:plugins:migrate VERSION=20170330131901 NAME=redmine_dmsf
```
4. Run the documents migration again:
```bash
bundle exec rake redmine:plugins:migrate
5. Watch the migration output and the production.log too.
```

**Comment by armandoazcano** [comment]
Hello 

I'm trying to move a DMSF from Redmine 3.2.0 and DMSF 1.5.5 to a new installation with redmine 3.4.2 and DMFS 1.6.0 (bitanmi debian8)

All looks fine but at the end all the DMSF files are unavailable.

The migration looks fine

----
Migrating to MigrateDocuments (20170418104901)
== 20170418104901 MigrateDocuments: migrating =================================
-- DmsfFile ID 1: project = nil
-- DmsfFile ID 2: project = nil
-- DmsfFile ID 3: project = nil
....
20170418104901 MigrateDocuments: migrated (0.7702s) ========================
----

But when the proccess finished,  the directory structure was not changed and the files are stored in the old folders.

When I try to download one of then the DMSF try to get th file from the new directory structure, but the files are in the old one.

I check all the folder a files permission and are ok

Can you help me ?

Thx in advance
Armando

**Comment by picman** [comment]
Hopefully solved in scope of #767.
